src/common/logger/ShortlistingLogger.ts

```
@@ -450,4 +442,404 @@ export class ShortlistingLogger {
+
+      // Check if file exists to determine if we need to write headers
+      const fileExists = fs.existsSync(logFilePath);
+
+      // Prepare CSV line with proper comma delimiters
+      const csvLine = [
+        ShortlistingLogger.escapeCSV(enrollmentFailureData.dateTime),
+        ShortlistingLogger.escapeCSV(enrollmentFailureData.userId),
+        ShortlistingLogger.escapeCSV(enrollmentFailureData.cohortId),
+        ShortlistingLogger.escapeCSV(enrollmentFailureData.courseId),
+        ShortlistingLogger.escapeCSV(enrollmentFailureData.failureReason),
+        ShortlistingLogger.escapeCSV(enrollmentFailureData.errorCode || "")
```

ESLint

[error] Insert `,`

(prettier/prettier)